.gitignore

```diff
@@ -66,6 +66,9 @@ node_modules/
 codekit-config.json
 .pycharm_helpers/
 
+.project
+.pydevproject
```

Comment:
ok, I't never that commit and than we can talk about the rest of the stuff in this gitignore